src/lib/components/dialog/QDialog.svelte

```diff
@@ -1,77 +1,58 @@
 <script lang="ts">
   import { clickOutsideDialog } from "$lib/helpers";
-  import { createClasses } from "$lib/utils";
-  import { createEventDispatcher, onMount } from "svelte";
+  import { createEventDispatcher } from "svelte";
   import { QBtn } from "$lib";
   import type { QDialogProps } from "./props";
 
-  export let noBtn: QDialogProps["noBtn"] = false,
-    btnContent: QDialogProps["btnContent"] = undefined,
-    btnAttrs: QDialogProps["btnAttrs"] = {},
+  export let button: QDialogProps["button"] = false,
+    buttonLabel: QDialogProps["buttonLabel"] = undefined,
+    buttonProps: QDialogProps["buttonProps"] = {},
     position: QDialogProps["position"] = "default",
     modal: QDialogProps["modal"] = false,
     fullscreen: QDialogProps["fullscreen"] = false,
     persistent: QDialogProps["persistent"] = false,
-    userClasses: QDialogProps["userClasses"] = undefined;
+    userClasses: QDialogProps["userClasses"] = undefined,
+    value: QDialogProps["value"] = false;
   export { userClasses as class };
 
   const emit = createEventDispatcher();
-  let dialogElement: HTMLDialogElement;
+  let dialogElement: HTMLDialogElement | undefined;
 
-  let opened = false;
+  const DURATION_TOGGLE = 150;
```

Comment:
Can we change this to `TOGGLE_DURATION` or something different like `ANIMATION_DURATION`?